message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1589,8 +1589,8 @@
       <description>Camera vision based attitude and position deltas.</description>
       <field name="time_usec" type="uint64_t" units="us">Timestamp (synced to UNIX time or since system boot).</field>
       <field name="time_delta_usec" type="uint64_t" units="us">Time since the last reported camera frame.</field>
-      <field type="float[3]" name="angle_delta">Defines a rotation vector in body frame that rotates the vehicle from the previous to the current orientation.</field>
-      <field type="float[3]" name="position_delta" units="m">Change in position from previous to current frame rotated into body frame (0=forward, 1=right, 2=down).</field>
+      <field type="float[3]" name="angle_delta" units="rad">Defines a rotation vector to the current MAV_FRAME_BODY_FRD from the previous MAV_FRAME_BODY_FRD.</field>
```

Comment:
I think the addition of the units is good but I'm less sure if the rest of the changes make it more or less clear.  I think knowing the axis order of the float[3] would be good but I don't think it is FRD (front, right, down) is it?  Surely it is roll, pitch, yaw or something isn't it?

Reply:
Roll, pitch, & yaw define the euler angles that rotate from for example Local_NED to the Body_FRD. I've added those as they certainly should be there.

x: Forward, y: Right, z: Down: Define the order, axis, and directionality of those rotations for a given Frame.